spec/classes/nftables_spec.rb

```diff
@@ -298,6 +310,31 @@
         it { is_expected.to have_nftables__set_resource_count(0) }
       end
 
+      context 'when purging unmanaged rules' do
+        let(:params) do
+          {
+            'purge_unmanaged_rules' => true,
+            'inmem_rules_hash_file' => '/foo/bar',
+          }
+        end
+
+        it { is_expected.not_to contain_file('/foo/bar') }
```

Comment:
you can probably please rubocop by running `bundle exec rake rubocop:autocorrect`

Reply:
Ran autocorrect, thanks.